auth/build.gradle

```diff
@@ -30,6 +30,13 @@ android {
     testOptions {
         unitTests.includeAndroidResources = true
     }
+
+    task generateAuthJavadoc(type: Javadoc){
```

Comment:
is there a similar task for other modules like core and metrics? I think this should be a top level task that generate javadocs for ALL the modules.

Reply:
I was running into problems getting it to work for all other modules, so I decided to put in the PR as this covers the scope of the ticket, and we could look at making it a top level task later on.